src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -1307,11 +1310,17 @@ private async Task ValidateUnusedIdsAsync(HashSet<string> unusedIds, string data
 
         }
 
-        public static unsafe bool IsBase64String(string base64)
+        public static unsafe bool IsBase64Id(string base64)
         {
```

Comment:
this should validate the length, it is expecting to get exactly 22 chars?

Reply:
Yes, Done